doc/articles/get-started-vs-2022.md

```diff
@@ -64,4 +64,8 @@ To setup your environment for WSL:
 
 ## Next Steps
 
-Now that you're done with setting up your environment, you can create your [first Uno Platform app](xref:Uno.GettingStarted.CreateAnApp.VS2022).
\ No newline at end of file
+You're all set! Now that you're done with setting up your environment.
+
+Learn more about:
```

Comment:
"You're all set! Now that you're done with setting up your environment, learn more about:"

Reply:
Made in PR https://github.com/unoplatform/uno/pull/14774